### ISSUE TITLE
Add user scripts aren't really workloads

### ISSUE THREAD

**Issue by crd477** [NVIDIA/deepops#839]
The 2 scripts in `workloads/examples/k8s/users`, `add_user.sh` and `k8s_user.sh` seem misplaced.  They're related to k8s administration, not really k8s workloads.

Also, I did find `add_user.sh` helpful as a source of hints for doing what I wanted in my own cluster but both scripts could really use some more comments and contextual documentation.

**Comment by supertetelman** [comment]
Yeah, you are correct. I think the "workloads" is a bit misleading and this is really workloads++.

This directory is meant to host example workloads, reference service stack (i.e. grafana/dcgm-exporter), and other templates that can be used as-is, but are just presented as some best practice examples. Overall, we are trying to keep the directory structure sensible, but not overloaded with many subfolders.

I have an open PR that adds a readme here adding a bit more of these details, hopefully that helps.

**Comment by crd477** [comment]
Might you consider combining the existing specimens into one better example and then moving it into `/scripts`?

**Comment by crd477** [comment]
Bumping in order to keep this open and as a reminder to @supertetelman that he mentioned a PR to add a README here.
